SR2024/2024-04-15-competition-wrapup.md

```diff
@@ -0,0 +1,64 @@
+
+Congratulations again to our winners:
+
+| Prize                           | Team
+|---------------------------------|--------------------------------
+| 1st Place                       | Gymnasium Markt Indersdorf
+| 2nd Place                       | Wisbech Grammar School
+| 3rd Place                       | The Ladies’ College, Guernsey
+| Rookie Award                    | The Henry Box School
+| Excellence in Engineering Award | Abingdon School
+| Robot and Team Image            | Gymnasium Markt Indersdorf
+| Challenges                      | Royal Grammar School Guildford
+| Online Presence                 | Westminster City School
+
+## Event Media
+
+If you'd like some mementos of the competition or would just like to see the photos which were taken, we've collected them all together into a photo album, which will be added to the blog post in the coming weeks.
+
+The livestreams of the matches are also available on YouTube, including footage from the final:
+
+- Day 1: https://www.youtube.com/watch?v=XhEUXg2m31k
+- Day 2: https://www.youtube.com/watch?v=CpCC2fTn0os
+
+The in-venue soundtrack playlists can also be found on [the event page](https://studentrobotics.org/events/sr2024/competition/#soundtrack).
+
+## Feedback
+
+We'd love to get feedback on how the competition was for you and your competitors this year, so we know what we did well, where we can improve, and to help apply for future funding.
+
+Please share the below short, anonymous survey with your competitors: [Send us feedback](https://forms.gle/88TS3hrJbvbjS2rEA)
+
+Additionally, we welcome feedback from you the supervisors too. Please fill in this survey: [Supervisor Feedback](https://forms.gle/ttRNrnwQwqM1wJWZ6)
+
+Without all of your invaluable feedback we wouldn't be able to run the competition and attract new sponsors.
+
+## Outstanding kit
+
+Whilst most of you will have returned your kit at the end of the competition event, you may still have the other pieces of Student Robotics kit around your college. If you were unable to attend the competition, this will likely be a full kit.
+
+If you do still have some of our kit, please let us know as soon as possible so we can organise its safe return. It is vitally important that all kit is returned to us so we can clean, test and re-issue it for future competitions.
+
+We will be contacting you soon if our records indicate you still have some items of kit.
+
+### Volunteering
+
+Student Robotics is powered by a collection of passionate individuals with varying skills from diverse backgrounds, many of whom are former competitors. We're all volunteers who give our spare time to make these events possible. If you know anyone who may be interested in volunteering, please ask them to sign up to be a blueshirt at [srobo.org/join](https://srobo.org/join). It's not all about software engineering - it takes everything from event planning to fundraising, marketing to project management to make these events possible.
```

Comment:
Why would they think it was all about software? Perhaps frame like "there's lots of stuff to do"?
```suggestion
Student Robotics is powered by a collection of passionate individuals with varying skills from diverse backgrounds, many of whom are former competitors. We're all volunteers who give our spare time to make these events possible. If you know anyone who may be interested in volunteering, please ask them to sign up to be a blueshirt at [srobo.org/join](https://srobo.org/join). As well as preparing the kit software & hardware for teams to use, it takes everything from event planning to fundraising, marketing to project management to make these events possible.
```